src/components/Asset/AssetLogo.tsx

```diff
@@ -57,7 +58,7 @@ export const AssetLogo: FC<AssetLogoProps> = ({ className, tokenAddress }) => {
         className
       )}
     >
-      {tokenAddress ? (
+      {isClientReady && tokenAddress ? (
```

Comment:
This can cause a hydration error when used outside the context of our current vaults.

Reply:
yes, also fixed it in the concentrator PR.